etc/profile-a-l/ani-cli.profile

```diff
@@ -0,0 +1,72 @@
+# Firejail profile for ani-cli
+# Description: Shell script to watch Anime from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ani-cli.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${DOWNLOADS}
+noblacklist ${HOME}/.cache/ani-cli
+noblacklist ${HOME}/.config/mpv
+noblacklist ${HOME}/.local/state/ani-cli
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/ani-cli
+mkdir ${HOME}/.local/state/ani-cli
+
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.cache/ani-cli
+whitelist ${HOME}/.config/mpv
+whitelist ${HOME}/.local/state/ani-cli
+
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+notv
+nou2f
+protocol unix,inet,inet6,netlink
+seccomp
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+private-bin ani-cli,aria2c,cat,cp,curl,cut,ffmpeg,fzf,grep,head,mkdir,mpv,mv,nl,sed,sh,sort,tput,tr,uname,wc
+private-cache
+private-dev
+private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,mime.types,nsswitch.conf,pango,pki,protocols,pulse,resolv.conf,rpc,services,ssl,X11,xdg
```

Comment:
There's a rework happening for private-etc (cfr. #5610). I suggest you keep this 'old' format commented and add the 'new' format just below it. That would be `private-etc @network,@sound,@tls-ca,@x11,mime.types`.

Reply:
Same note as in the following comment:

* <https://github.com/netblue30/firejail/pull/5706#discussion_r1124284952>

Reply:
Got it! Same reply as in https://github.com/netblue30/firejail/pull/5706#discussion_r1124297337.